refinery/tool_manager/utils.py

```diff
@@ -11,3 +19,175 @@ class AdminFieldPopulator(admin.ModelAdmin):
     def __init__(self, model, admin_site):
         super(AdminFieldPopulator, self).__init__(model, admin_site)
         self.list_display = [field.name for field in model._meta.fields]
+
+
+class WorkflowAnnotationValidationError(Exception):
```

Comment:
See: https://docs.python.org/2/library/exceptions.html#exceptions.RuntimeError

For removal of FileTypeValidationError & WorkflowValidationError